### PR TITLE
Cleanup Todos in input.fs

### PR DESCRIPTION
**Pull request by jjohnson338**
Took a swing at cleaning up a couple of TODOs found in the `input.fs` file.

### CODE REVIEW

File: input.fs [hunk 1]
```diff
@@ -218,8 +218,7 @@ let (|Mouse|Special|Normal|ImeEvent|TextInput|Unrecognized|) (x: InputEvent) =
     |  Key(_, Key.Add)                                            -> Special("kPlus")
     |  Key(_, Key.Subtract)                                       -> Special("kMinus")
     |  Key(_, Key.Divide)                                         -> Special("kDivide")
-    // TODO|  Key(_, Key.Decimal) -> ???
```
Comment (jjohnson338) [inline input.fs:221]: This looks to be covered by line 187.

Comment (yatli) [reply input.fs:221]: must've been abandoned in the first run, then added later ;)

File: input.fs [hunk 2]
```diff
@@ -218,8 +218,7 @@ let (|Mouse|Special|Normal|ImeEvent|TextInput|Unrecognized|) (x: InputEvent) =
...
-    // TODO|  Key(_, Key.Separator) -> ???
+    |  Key(_, Key.Separator)                                      -> Special("Bar")
```
Comment (yatli) [inline input.fs:221]: this one I'm not sure.
see: https://github.com/AvaloniaUI/Avalonia/search?q=Key.Separator

**Review by yatli — CHANGES_REQUESTED** [review]

File: input.fs (hunk 2 above)
Comment (yatli) [reply input.fs:221]: according to this thread: https://stackoverflow.com/questions/36666868/uk-keyboard-virtual-key-code-for-the-symbol
`Key.Separator` is the "Enter" key on a numpad and I don't even have a numpad to test it 😂
Maybe map to `<kEnter>`.

**Comment by yatli** [comment]
and thanks for cleaning this up!

File: input.fs (hunk 2 above)
Comment (yatli) [reply input.fs:221]: and maybe `Key.Decimal` to `<kPoint>`?

**Comment by yatli** [comment]
@jjohnson338 I've also cleaned up the keypad mappings (0efa44b)

**Pull request merged**